ecpy_hqc_legacy/instruments/drivers/driver_tools.py

```diff
@@ -135,8 +135,9 @@ def wrapper(self, *args, **kwargs):
                         raise
                     else:
                         log = logging.getLogger(__name__)
-                        msg = 'Iterating connection %s/%s' % (i, max_iter)
-                        log.exception(msg)
+                        msg = 'Iterating connection %s/%s ' % (i, max_iter)
```

Comment:
This could be written more cleanly as:
```python
msg = ('Iterating connection %s/%s '
            'for instrument %s')
log.debug(msg, (i, max_iter, type(self).__name__)
```
This has the added advantage that the message will be formatted only if a logger actually handle it (here we don't care but just so that you know). Furthermore the PyVisa driver will not help the user to know what is going on so here I pass the name of the class so that at least it can know what kind of instrument is behaving badly.